docs/components/segmented-control.md

```diff
@@ -0,0 +1,102 @@
+# Segmented Control
+
+**KSegmentedControl** is used a like radio button group and is meant to toggle between mutually exclusive options.
+
+<KToggle v-slot="{ isToggled, toggle }">
+  <KSegmentedControl
+      :options="['On','Off']"
+      :selected="isToggled?'On':'Off'"
+      @toggled="toggle"
+  />
+</KToggle>
+
+```vue
+<KToggle v-slot="{ isToggled, toggle }">
+  <KSegmentedControl
+      :options="['On','Off']"
+      :selected="isToggled?'On':'Off'"
+      @toggled="toggle"
+  />
+</KToggle>
+```
+
+## Props
+
+### options - required
+
+An array of options for each button, can also be provided as a json object in order to use a custom label.
+
+<KToggle v-slot="{ isToggled, toggle }">
+  <KSegmentedControl
+      :options="[{label:'◀️',value:'left'},{label:'▶️',value:'right'}]"
+      :selected="isToggled?'left':'right'"
+      @toggled="toggle"
+  />
+</KToggle>
+
+```vue
+<KToggle v-slot="{ isToggled, toggle }">
+  <KSegmentedControl
+      :options="[{label:'◀️',value:'left'},{label:'▶️',value:'right'}]"
+      :selected="isToggled?'left':'right'"
+      @toggled="toggle"
+  />
+</KToggle>
+```
+
+### selected
```

Comment:
we should use v-model instead so it's more consistent with our other components and is more vue-like for consumers

Reply:
looks like selected is not a prop anymore. This can be used to be "v-model" now.